Add open spaces page

marinamorolopez opened the pull request:
This PR adds a new page for Open Spaces (based on the existing Sprints page) with the main info and the digitalized system to make bookings

<!-- readthedocs-preview ep-website start -->
🖼️ Preview available 🖼️ : https://ep-website--1360.org.readthedocs.build/
<!-- readthedocs-preview ep-website end -->

marinamorolopez committed:
Add Open Spaces page

marinamorolopez committed:
Merge branch 'EuroPython:ep2025' into add-open-spaces-page

pre-commit-ci[bot] committed:
[pre-commit.ci] auto fixes from pre-commit.com hooks

for more information, see https://pre-commit.ci

hypha commented on src/content/pages/open-spaces.md:
```diff
@@ -0,0 +1,61 @@
+Check the
+[Open Spaces schedule ↗️!](https://calendar.google.com/calendar/u/0?cid=Y19iODcwNzViMjljZjRhOWRkNGRhM2ZlNzI0ZmNiYTkyYTRiNDA2YWEzZjkxY2I3ZTdjZTUwZmVkZTQzMjE1YjE3QGdyb3VwLmNhbGVuZGFyLmdvb2dsZS5jb20)
+
+## How to Organize an Open Space
```
```suggestion
## How to Organise an Open Space
```
maybe use UK spelling to be consistent?

hypha commented on src/content/pages/open-spaces.md:
```diff
@@ -0,0 +1,61 @@
+
+## How to Organize an Open Space
+
+To organize your own Open Space session, simply book a free time slot and room
```
```suggestion
To organize your own Open Space session, simply book a free time slot and room
```
maybe use UK spelling to be consistent?

hypha commented on src/content/pages/open-spaces.md:
```diff
@@ -0,0 +1,61 @@
+- Community building and diversity in tech
+- Open source contributions
+- Mentoring
+- Career growth and personal development
```
Like Mia pointed out, good to add a few non-coding related stuff, e.g. knitting, building ham radios, dancing, etc so it's clear that those are also highly encouraged.

hypha commented on src/content/pages/open-spaces.md:
```diff
@@ -0,0 +1,61 @@
+presentation.
+
+Both Room 221+222 and Room 223+224 will be reserved exclusively for Open Space
+sessions. You can book either space by using the links below:
```
Is it worth mentioning that South Room 221 + 222 has tables and a 65'' monitor; whereas Room 223+224 doesn't have tables (just chairs) and doesn't have the monitor? This might be relevant for people when they book?

hypha reviewed (commented):
Thank you so much for the PR. I added a few small comments. Some that are not in the diff: 
1. It'd be nice to link it both in the menu and the footer: https://github.com/marinamorolopez/website/blob/add-open-spaces-page/src/data/links.json I imagine it could be under events? 
2. It should probably be listed here: https://github.com/marinamorolopez/website/blob/ccc2506fa646a84ca8319f3f7d2209189398e75f/src/pages/overview.astro#L140 
3. Because the wonderful effort to digitalise the signup, this bit is now out of date: 
https://github.com/marinamorolopez/website/blob/ccc2506fa646a84ca8319f3f7d2209189398e75f/src/content/pages/faq.mdx?plain=1#L132. Is it possible to change it to something like " Simply book a free time slot and room via our Open Spaces registration calendar, which also serves as a notice board to attract interested attendees.

marinamorolopez committed:
Merge branch 'EuroPython:ep2025' into add-open-spaces-page

marinamorolopez committed:
Update open-spaces.md

Update the open space page to add non-technical topics, use British spelling in “organize,” specify technical aspects of open space rooms, and provide an alternative way to reserve slots for people who do not have a Google account

pre-commit-ci[bot] committed:
[pre-commit.ci] auto fixes from pre-commit.com hooks

for more information, see https://pre-commit.ci

marinamorolopez committed:
Add Open Spaces link to menu and footer

Open Spaces link added to menu and footer under Events

marinamorolopez committed:
Add Open Spaces page to Programme Overview

Open Spaces link added to the Programme - Overview page under 'Open Spaces'

marinamorolopez committed:
Update Open Spaces info in FAQ

Updated the Open Spaces booking system from 'Open Space signup board' to 'Open Spaces Registration calendar'

hypha reviewed (approved):
Thank you for creating the beautiful page, Marina, and for addressing all the comments! <3

clytaemnestra reviewed (approved):
LGTM, thanks a lot!

clytaemnestra committed:
Merge branch 'ep2025' into add-open-spaces-page
